src/status_im/chat/models/mentions.cljs

```diff
@@ -163,25 +162,12 @@
               :match         match
               :searched-text searched-text
               :public-key    public-key})
-  (let [{:keys [new-text state chat-id]} (transfer-mention-result result)
-        {:keys [at-sign-idx]}            state
-        cursor                           (+ at-sign-idx (count primary-name) 2)]
+  (let [{:keys [new-text chat-id]} (transfer-mention-result result)]
     (rf/merge
      cofx
-     (let [common {:db       (-> db
-                                 (assoc-in [:chats/mention-suggestions chat-id] nil))
-                   :dispatch [:chat.ui/set-chat-input-text new-text chat-id]}
-           extra  (if (not config/new-composer-enabled?)
-                    ;; NOTE(rasom): Some keyboards do not react on selection property passed to
-                    ;; text input (specifically Samsung keyboard with predictive text set on).
-                    ;; In this case, if the user continues typing after the programmatic change,
-                    ;; the new text is added to the last known cursor position before
-                    ;; programmatic change. By calling `reset-text-input-cursor` we force the
-                    ;; keyboard's cursor position to be changed before the next input.
-                    {:set-text-input-value    [chat-id new-text text-input-ref]
```

Comment:
not sure we won't have the same issue(comments deleted above) when using new composer? if still exist, we should keep `reset-text-input-cursor`?

Reply:
It is handled in a different way in the new composer